packages/asset-service/src/service/descriptions/index.ts

```diff
@@ -0,0 +1,14 @@
+import en from './en.json'
+import es from './es.json'
+import fr from './fr.json'
+import id from './id.json'
+import ko from './ko.json'
+import pt from './pt.json'
+import ru from './ru.json'
+import zh from './zh.json'
+
+const descriptions = { en, es, fr, id, ko, pt, ru, zh } as {
+  [locale: string]: Record<string, string>
+}
+
+export default descriptions
```

Comment:
not sure the need for the typing here
```suggestion
export default { en, es, fr, id, ko, pt, ru, zh }
```

Reply:
This will type the JSON (which are always typed as any) for consumers:

<img width="456" alt="image" src="https://user-images.githubusercontent.com/17035424/176734337-018bf2bc-d08e-4e4b-b4d8-5a207746929d.png">

We currently do so by creating a new typed variable https://github.com/shapeshift/lib/pull/849/files#diff-7ac6886bda84d5c7853bc86947fb6253944e449aaef53bf61b96a656690b236bL25 but we can now do the same directly at export-level

Reply:
I tested locally and it is inferring types correctly for me which is why I commented. hmm..

Reply:
localeDescriptions is any though I am noticing in the asset service since local is just a string and can't be guaranteed from the inferred type

Reply:
It's fine, we can leave as is makes sense to avoid ambiguity

Reply:
Yeah, if that was a JS file we could `as const` it and better type `locale` to properly discriminate depending on the accessor, but as this is a JSON, this is the best typings we can get for now!